workshop/docker/data/README.md

```diff
@@ -8,12 +8,22 @@ This directory provides sample data to demonstrate functionality.
 - title: [free-wifi-florence.csv](free-wifi-florence.csv)
 - url: https://opendata.comune.fi.it/?q=metarepo/datasetinfo&id=fb5b7bac-bcb0-4326-9388-7e3f3d671d71
 - description: The dataset shows the location of the places in the Municipality of Florence where a free wireless internet connection service (Wifi) is available.
-- copyright: Comune di Firenze 
+- copyright: Comune di Firenze
 - license: CC-BY 4.0
 
-### source 2
-- title: TODO
-- url: TODO
-- description: TODO
-- copyright: TODO
-- license: TODO
+
+## Metadata/Records
+
+### Free Wifi Hotspots in Florence
+- title: [free-wifi-florence.xml](./metadata/xml/free-wifi-florence.xml)
+- url: https://opendata.comune.fi.it/?q=metarepo/datasetinfo&id=fb5b7bac-bcb0-4326-9388-7e3f3d671d71
+- description: The dataset shows the location of the places in the Municipality of Florence where a free wireless internet connection service (Wifi) is available.
+- copyright: Comune di Firenze
+- license: CC-BY 4.0
+
+### Carsharing spots in Florence
+- title: [carsharing-florence.xml](./metadata/xml/carsharing-florence.xml)
```

Comment:
Should this also be ` - title: [carsharing-florence.xml](./records/xml/carsharing-florence.xml)` ?